var/spack/repos/builtin/packages/py-kosh/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyKosh(PythonPackage):
+    """Kosh allows codes to store, query, share data via an easy-to-use Python API.
+Kosh lies on top of Sina and as a result can use any database backend supported by Sina.
+In adition Kosh aims to make data access and sharing as simple as possible.
+"""
```

Comment:
I would indent this 4 spaces

Reply:
```suggestion
    Kosh lies on top of Sina and as a result can use any database backend supported by Sina.
    In adition Kosh aims to make data access and sharing as simple as possible.
    """
```
May need to wrap some of these lines if they are too long.